store/postgres/src/store.rs

```diff
@@ -57,7 +60,7 @@ pub struct Store {
     logger: slog::Logger,
     subscriptions: Arc<RwLock<HashMap<String, Subscription>>>,
     change_listener: EntityChangeListener,
-    pub conn: PgConnection,
+    pub conn: Arc<Mutex<PgConnection>>,
```

Comment:
Curious: Why is this needed?

Reply:
Oh, nevermind. I see you need the collection in the async parts of the `BlockStore` implementation.